plugins/modules/win_gpo_force.ps1

```diff
@@ -0,0 +1,75 @@
+#!powershell
+# Copyright: (c) 2021 Sebastian Gruber (@sgruber94) ,dacoso GmbH All Rights Reserved.
+# SPDX-License-Identifier: GPL-3.0-only
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+$spec = @{
+    options             = @{
+        mode = @{ type = "str"; choices = "sysvolonly", "forceupdate"; required =$true}
+        ou   = @{ type = "str"; aliases = @("organizational_unit") }
+    }
+    supports_check_mode = $false
+}
+$module = [Ansible.Basic.AnsibleModule]::Create($args, $spec)
+$mode = $module.Params.mode
+$ou = $module.Params.ou
+try {
+    Import-Module GroupPolicy
+} catch {
+    $module.FailJson("win_gpo_force requires the GroupPolicy PS module to be installed")
+}
+function Invoke-forcedgpo {
+    param(
+        [string]$oucomputer	# Push gpupdate after AD & SYSVOL Replication
+    )
+    # Check for DFSR cmd tools
+    if ((Get-WindowsFeature -Name RSAT-DFS-Mgmt-Con).InstallState -ne "Installed") {
+        Install-WindowsFeature -Name RSAT-DFS-Mgmt-Con
+    }
+    # Check powershell console for UAC
+    #requires -RunAsAdministrator
+    # Force Replication of all Partitions between Domain Controllers
+    (Get-ADDomainController -filter *).Name | ForEach-Object { repadmin /syncall $_ (Get-ADDomain).DistinguishedName /e /A | Out-Null }
```

Comment:
You use this twice, better off storing the list of AD controllers into a var then doing `$controllers | ForEach-Object ...` in both sections instead of running the query twice.